ontotext-yasgui-web-component/src/services/yasr/yasr-service.ts

```diff
@@ -0,0 +1,111 @@
+export class YasrService {
+
+  static disablePlugin(name: string) {
+    this.setPluginEnable(name, false);
+  }
+
+  static enablePlugin(name: string) {
+    this.setPluginEnable(name, true);
+  }
+
+  static setPluginEnable(name: string, enable: boolean) {
+    // @ts-ignore
+    if (Yasr.defaults.plugins) {
+      // @ts-ignore
+      const plugin = Yasr.defaults.plugins[name];
+      if (plugin) {
+        plugin.enabled = enable;
+      }
+    }
+  }
+
+  static getPluginsConfigurations(): Map<string, any> {
+    const pluginsConfigurations = new Map<string, any>();
+    pluginsConfigurations.set('extended_table', YasrService.getExtendedTableConfiguration());
+    return pluginsConfigurations;
+  }
+
+  private static getExtendedTableConfiguration() {
+    return {
+      getCellContent: YasrService.getCellContent.bind(this)
```

Comment:
To be honest, I'd prefer these plugin implementations to be in their own classes and files and not to mess with our services. You can leave it this way now, but we should consider extracting them.